sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/KeyAsyncClient.java

```diff
@@ -1435,7 +1439,7 @@ Duration getDefaultPollingInterval() {
     }
 
     /**
-     * Releases a key.
+     * Release the latest version of a key.
```

Comment:
```suggestion
     * Releases the latest version of a key.
```
To match the voice used in other methods.